cmd/build.go

```diff
@@ -603,3 +607,19 @@ func mapLinkedVars(mapping map[string]interface{}, manifestPath string, config *
 
 	return nil
 }
+
+// HumanizeKustomizeError takes errors returned from GenerateKustomizeResult and returns them in a human friendly string
+func HumanizeKustomizeError(err error) string {
```

Comment:
Should this be `HumanizeKustomizeNamespaceError` since the error messages are namespace specific?

Reply:
We could do that. I figured it could expand to other errors. This is specific to the `GenerateKustomizeResult` function. So maybe we should call it `HumanizeKustomizeResultError`?